src/main/java/com/gamingmesh/jobs/listeners/JobsPaymentListener.java

```diff
@@ -380,14 +380,13 @@ public void onBlockBreak(BlockBreakEvent event) {
 	// Checks whether or not the plant is ripe if it does not get money at that time
 	if (block.getType().equals(CMIMaterial.WHEAT_SEEDS) || block.getType().equals(CMIMaterial.BEETROOT_SEEDS) ||
 			block.getType().equals(CMIMaterial.MELON_SEEDS) || block.getType().equals(CMIMaterial.WHEAT_SEEDS) ||
-			block.getType().equals(CMIMaterial.PUMPKIN_SEEDS) || block.getType().equals(CMIMaterial.CARROT) ||
+			block.getType().equals(CMIMaterial.PUMPKIN_SEEDS) || block.getType().toString().equals("CARROTS") ||

```

Comment:
Zrips fixed the carrot recogniting bug in [this commit](https://github.com/Zrips/Jobs/commit/ac31db0927384f66c1862582c5e91081f86bedbf#diff-2bbefed2dca880e596848b248efa0102L737).

Reply:
Yep, but there's still a duplication bug with this change.. Did he try before to close ticket ?